### DISCUSSION TITLE
Sumatra 3.5.2 allways opens last document in  a new window

### DISCUSSION THREAD

**Discussion by MaxJalon** [sumatrapdfreader/sumatrapdf#4325] [General]
Installed Sumatra 3.5.2 - 64 Windows today. 

I have tabs switched of but history on. 

When I double click a PDF document it opens, but also a second window with the last PDF in the "recent" list.

How can I stop that?

**Comment by GitHubRulesOK** [comment]
Tabs off means multiple single window instances and if remember last session is on then that last file will occupy a second window it is controlled by `RestoreSession = true` so setting that `RestoreSession = false` should work

**Comment by MaxJalon** [comment]
That did it, thank you very much!

**Comment by kjk** [comment]
Also this should be fixed in latest https://www.sumatrapdfreader.org/prerelease even without disabling sessions
